### ISSUE TITLE
No changelog for Unreleased version

### ISSUE THREAD

**Issue by pshevtsov** [git-chglog/git-chglog#15]
I see no changelog for Unreleased version, however, there are untagged commits:

```console
$ git log
commit 143ee38b33ed57c5e16bf040e58eed1f312f58a2 (HEAD -> master)
Author: pshevtsov <petr.shevtsov@gmail.com>
Date:   Thu Apr 5 17:53:16 2018 +0300

    feat: hello

commit 344e2d3accee00c4d91c23e5c60b0f6f5cb24d11 (tag: v0.0.3)
Author: pshevtsov <petr.shevtsov@gmail.com>
Date:   Thu Apr 5 17:25:15 2018 +0300

    feat: saying goodbye

    I decided to say goodbye, I don't want to greet anyone anymore.

commit 0257adb5d9790320295e29c7d5d842df29ef8a76 (tag: v0.0.2)
Author: pshevtsov <petr.shevtsov@gmail.com>
Date:   Thu Apr 5 17:22:40 2018 +0300

    fix: corrected greeting

commit d5eb564e98513c1a61a14f950317e3a4e5827a5c (tag: v0.0.1)
Author: pshevtsov <petr.shevtsov@gmail.com>
Date:   Thu Apr 5 17:20:11 2018 +0300

    feat: initial commit
```

## Expected Behavior

```console
$ git-chglog
## [Unreleased]
### Features
- hello

## v0.0.3 - 2018-04-05
### Features
- saying goodbye

## v0.0.2 - 2018-04-05
### Bug Fixes
- corrected greeting

## v0.0.1 - 2018-04-05
### Features
- initial commit
```

## Actual Behavior

```console
$ git-chglog
## [Unreleased]

## v0.0.3 - 2018-04-05
### Features
- saying goodbye

## v0.0.2 - 2018-04-05
### Bug Fixes
- corrected greeting

## v0.0.1 - 2018-04-05
### Features
- initial commit
```

## Steps to Reproduce (including precondition)

1. Commit to master, do not `git tag`
2. Run `git-chglog`

## Your Environment

- OS: Debian GNU/Linux unstable (sid) x86_64
- git-chglog version: 0.3.1

**Comment by tsuyoshiwada** [comment]
Hi @pshevtsov, Thank you for using `git-chglog`!

This behavior is not a bug.
It is output conforming to the specification of [keep-a-changelog](https://keepachangelog.com/en/1.0.0/) :+1:

**Comment by ringohub** [comment]
I have same problem. I want to display 'UNRELEASED' section at the changelog before tagging.

At 'keep-a-changelog' has 'UNRELEASED' section. For example: [keep-a-changelog/CHANGELOG.md Tag v0.3.0](https://github.com/olivierlacan/keep-a-changelog/blob/v0.3.0/CHANGELOG.md). But now can't scooping those commit logs. 🤔

**Comment by tsuyoshiwada** [comment]
@pshevtsov @ringohub 
I'm sorry, it was my misunderstanding ...

As @pshevtsov wrote, the behavior of `Expected Behavior` seems to be correct. 
I will try fix it :pray:

Thanks.

**Comment by tsuyoshiwada** [comment]
@pshevtsov @ringohub

Fixed it so that Unreleased commit can be added, and released it as `0.5.0` :tada:
`Unreleased` will be displayed by re-creating the template file with the` --init` command.

Please check it, if you have time. Thanks :dog:
